std/node/process.ts

```diff
@@ -1,32 +1,109 @@
 // Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
 import { notImplemented } from "./_utils.ts";
+import EventEmitter from "./events.ts";
+import { fromFileUrl } from "../path/mod.ts";
+
+const notImplementedEvents = [
+  "beforeExit",
+  "disconnect",
+  "message",
+  "multipleResolves",
+  "rejectionHandled",
+  "SIGBREAK",
+  "SIGBUS",
+  "SIGFPE",
+  "SIGHUP",
+  "SIGILL",
+  "SIGINT",
+  "SIGSEGV",
+  "SIGTERM",
+  "SIGWINCH",
+  "uncaughtException",
+  "uncaughtExceptionMonitor",
+  "unhandledRejection",
+  "warning",
+];
 
 /** https://nodejs.org/api/process.html#process_process_arch */
 export const arch = Deno.build.arch;
 
+function getArguments() {
+  return [Deno.execPath(), fromFileUrl(Deno.mainModule), ...Deno.args];
+}
+
+//deno-lint-ignore ban-ts-comment
+//@ts-ignore
```

Comment:
@Soremwar FYI you can remove `deno-lint-ignore` if you add a comment to `// @ts-ignore` on why it's being used